samples/multiplatform/settings.gradle

```diff
@@ -1,4 +1,9 @@
+enableFeaturePreview('GRADLE_METADATA')
```

Comment:
Is that needed for the sample?
Then we have to documented that as well

Reply:
The sample in a separate repo uses KN 0.7 and there is no need to add this line for that version.